src/ccalls.jl

```diff
@@ -15,6 +15,35 @@ function EvalStringEx(cmd::String)
     return res
 end
 
+"""
+    EvalString(cmd::String)
+
+Let GAP execute the command(s) given by `cmd` and returns the last result.
+An error is thrown if no value is returned.
+
+# Examples
+```jldoctest
+julia> GAP.EvalString( "1+2" )
+3
+
+julia> GAP.EvalString( "x:= []; y:= 0" )
+0
+
+julia> GAP.EvalString( "Add( x, 1 )" )
+ERROR: Error thrown by GAP: Error, List Element: <list>[2] must have an assigned value
+[...]
+
+julia> GAP.EvalString( "x:= []" )
+GAP: [  ]
+
+julia> GAP.EvalString( "Add( x, 1 ); 0" )
+0
```

Comment:
This one is a bit weird and against the original intention of `EvalString`. But I am fine with accepting it, but I count this as yet another motivation to resolve issue #474 (one shouldn't have to add a trailing `0` like this to execute an `Add`)